msal/src/main/java/com/microsoft/identity/client/AuthenticationResult.java

```diff
@@ -51,6 +52,19 @@ public AuthenticationResult(final ICacheRecord cacheRecord) {
         mAccount = AccountAdapter.adapt(cacheRecord.getAccount());
     }
 
+    public AuthenticationResult(AccessTokenRecord accessToken,
+                                String tenantId,
+                                String uniqueId,
```

Comment:
Careful.... what is this value?

If it's not the `<uid>.<utid>` it's not going to be the same as the `home_account_id` coming from the `AccountRecord`.

I suspect this value should be sourced from the `AccessTokenRecord` and can be removed as a param